src/index.ts

```diff
@@ -157,6 +163,22 @@ async function runUpgradeScripts(targetVersion: number) {
     }
 }
 
+async function runEslintFix() {
+    console.info("Fixing ESLint errors");
+
+    for (const microservice of microservices) {
+        if (!microserviceExists(microservice)) {
+            continue;
+        }
+
+        try {
+            await executeCommand("npm", ["run", "--prefix", microservice, "--no-audit", "--loglevel", "error", "lint:eslint", "--", "--fix"]);
+        } catch (e) {
+            console.error(`Error trying to run 'lint:eslint' in ${microservice}`);
```

Comment:
```suggestion
            console.error(`Failed to fix ESLint errors in ${microservice}. See original error below`);
            console.error(error);
```

I'd use a similar error like the one during the upgrade scripts runs